internal/acceptance/lakeview_dashboard_test.go

```diff
@@ -0,0 +1,412 @@
+				return nil
+			},
+			func(s *terraform.State) error {
+				dashboard, ok := s.RootModule().Resources["databricks_lakeview_dashboard.d1"]
+				if !ok {
+					return fmt.Errorf("dashboard resource not found")
+				}
+				dashboard_id = dashboard.Primary.ID
+				assert.NotEmpty(t, dashboard_id)
+				display_name = dashboard.Primary.Attributes["display_name"]
+				assert.NotEmpty(t, display_name)
+				warehouse_id = dashboard.Primary.Attributes["warehouse_id"]
+				assert.NotEmpty(t, warehouse_id)
+				etag = dashboard.Primary.Attributes["etag"]
+				assert.NotEmpty(t, etag)
+				return nil
+			},
+		),
+	}, step{
+		PreConfig: func() {
+			w, err := databricks.NewWorkspaceClient(&databricks.Config{})
+			if err != nil {
+				fmt.Println(err)
+			}
+			_, err = w.Lakeview.Update(context.Background(), dashboards.UpdateDashboardRequest{
+				DashboardId:         dashboard_id,
+				DisplayName:         display_name,
+				Etag:                etag,
+				WarehouseId:         warehouse_id,
+				SerializedDashboard: "{\"pages\":[{\"name\":\"b532570b\",\"displayName\":\"New Page Modified\"}]}",
+			})
+			if err != nil {
+				fmt.Println(err)
+			}
+		},
+		Template: `
+		resource "databricks_lakeview_dashboard" "d1" {
+			display_name			= 	"Monthly Traffic Report"
+			warehouse_id			=	"{env.TEST_DEFAULT_WAREHOUSE_ID}"
+			serialized_dashboard	=	"{\"pages\":[{\"name\":\"new_name\",\"displayName\":\"New Page\"}]}"
+			parent_path				= 	"/Shared/provider-test"
+		}
+		`,
+		Check: resource.ComposeTestCheckFunc(
+			resource.TestCheckResourceAttr("databricks_lakeview_dashboard.d1", "display_name", "Monthly Traffic Report"),
+			func(state *terraform.State) error {
+				dashboard, ok := state.RootModule().Resources["databricks_lakeview_dashboard.d1"]
+				if !ok {
+					return fmt.Errorf("dashboard resource not found")
+				}
+				fmt.Println(dashboard.Primary.Attributes["serialized_dashboard"])
+				return nil
+			},
+		),
+	})
+}
+
+func TestAccDashboardTestAll(t *testing.T) {
+	dashboard_id := ""
+	display_name := ""
+	warehouse_id := ""
+	etag := ""
+	workspaceLevel(t, step{
+		Template: `
+		resource "databricks_lakeview_dashboard" "d1" {
+			display_name			= 	"Monthly Traffic Report"
+			warehouse_id			=	"{env.TEST_DEFAULT_WAREHOUSE_ID}"
+			serialized_dashboard	=	"{\"pages\":[{\"name\":\"new_name\",\"displayName\":\"New Page\"}]}"
+			parent_path				= 	"/Shared/provider-test"
+		}
+		`,
+		Check: resource.ComposeTestCheckFunc(
+			resource.TestCheckResourceAttr("databricks_lakeview_dashboard.d1", "display_name", "Monthly Traffic Report"),
+			func(state *terraform.State) error {
+				dashboard, ok := state.RootModule().Resources["databricks_lakeview_dashboard.d1"]
+				if !ok {
+					return fmt.Errorf("dashboard resource not found")
+				}
+				fmt.Println(dashboard.Primary.Attributes["serialized_dashboard"])
+				return nil
+			},
+		),
+	}, step{
+		PreConfig: func() {
+			tmpDir := "/tmp/LakeviewDashboardTest"
```

Comment:
I suggest randomize these tmpDirs, for example https://github.com/databricks/terraform-provider-databricks/blob/75236a645b7d86819e43281e6c2623bbc5527528/internal/acceptance/cluster_test.go#L13

Reply:
Helper to create random things
https://github.com/databricks/terraform-provider-databricks/blob/main/qa/testing.go#L50